fitbenchmarking/fitbenchmark_one_problem.py

```diff
@@ -0,0 +1,119 @@
+"""
+Fit benchmark one problem functions.
+"""
+# Copyright &copy; 2016 ISIS Rutherford Appleton Laboratory, NScD
+# Oak Ridge National Laboratory & European Spallation Source
+#
+# This file is part of Mantid.
+# Mantid is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# Mantid is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# File change history is stored at:
+# <https://github.com/mantidproject/fitbenchmarking>.
+# Code Documentation is available at: <http://doxygen.mantidproject.org>
+
+
+from __future__ import (absolute_import, division, print_function)
+
+import sys
+import numpy as np
+import mantid.simpleapi as msapi
+
+
+from fitting import fit_algorithms, mantid_utils
+from fitting.plotting import plots
+from utils import fit_misc, test_result
+from utils.logging_setup import logger
+
+MAX_FLOAT = sys.float_info.max
+
+
+def fitbm_one_problem(prob, minimizers, use_errors=True, group_results_dir=None):
```

Comment:
Please explain what this method is doing. E.g. it is not immediately obvious how it differ from fitbm_one_problem_def